azure-pipelines.yml

```diff
@@ -681,8 +681,8 @@ jobs:
     displayName: 'Test Homebrew Package'
 
 
-- job: BuildYumPackageMariner
-  displayName: Build Yum Package Mariner
+- job: BuildRpmPackageMariner
```

Comment:
RPM is the actual package and YUM is the package manager (see https://packages.microsoft.com/yumrepos/azure-cli/). YUM has been replaced by DNF in many modern systems. So RPM should be the correct name here, like DEB (package) vs APT (package manager).